projects/kit/components/input-phone-international/input-phone-international.template.html

```diff
@@ -31,6 +31,7 @@
             [pseudoInvalid]="pseudoInvalid"
             [pseudoFocused]="pseudoFocused"
             [pseudoHovered]="pseudoHovered"
+            (paste.capture.prevent.stop)="onPaste($event)"
```

Comment:
There's also drop event. Probably works similarly. Also we can move it to HostListener to make template simpler.